src/logical/chrono.rs

```diff
@@ -1,4 +1,5 @@
-use chrono::{DateTime, NaiveDateTime, NaiveTime, Timelike, Utc};
+use arrow_array::temporal_conversions::NANOSECONDS;
```

Comment:
I appreciate that you tried to reuse this const, however, `arrow-array` is an optional dependency and not required when just building with the `chrono` module. So I'm fine with just adding a similar const to this module.

Reply:
Huh, I didn't notice that it was an auto-import at all. Will address.